app/src/androidTest/java/ch/epfl/sweng/runpharaa/TrackPropertiesActivityTest.java

```diff
@@ -192,8 +215,13 @@ public void testAddingCorrectComment(){
         sleep(3000);
     }
 
+    @Test
+    public void testDeletedButtonVisibility(){
```

Comment:
Mmmmm maybe delete this empty test ^^
(and testing the visibility of a button is a bit useless if you did not change its visibility in the code)